README.md

```diff
@@ -310,6 +325,54 @@ function assignId (req, res, next) {
 }
 ```
 
+#### arguments to custom token formats
+
+Example of a custom token format that uses an argument. It will output the
```

Comment:
I mean, if we are struggling to even come up with a real example at all, then why even make an example?